app/Http/Controllers/Backend/BrouterController.php

```diff
@@ -91,7 +91,7 @@ public static function reroutePolyline(HafasTrip $trip): void {
             //2. Request route at brouter
             $brouterGeoJSON = self::getGeoJSONForRoute($coordinates);
         } catch (InvalidArgumentException) {
-            Log::error('[RefreshPolyline] Error while getting Polyline for HafasTrip#' . $trip->trip_id . ' (Required data is missing in Brouter response)');
+            Log::warning('[RefreshPolyline] Error while getting Polyline for HafasTrip#' . $trip->trip_id . ' (Required data is missing in Brouter response)');
```

Comment:
I think error is correct here.

Reply:
Depends on how you're looking at it. This error is caused by outside dependencies. We cannot change the result of brouters response. If they can't find a route, I don't think that this should be an error because we can't fix it.